spinnman/transceiver.py

```diff
@@ -1266,6 +1282,10 @@ def set_watch_dog(self, watch_dog):
     def get_iobuf_from_core(self, x, y, p):
         """ Get the contents of IOBUF for a given core
 
+        This method is currently deprecated and untested as there is no
+        known use. Same functionaility provided by ybug and bmpc.
+        Retained in case needed for hardware debugging.
+
```

Comment:
This could be fully deleted as it is only a utility method anyway i.e. you can do the same thing with a different method and a little bit of extra work.

Reply:
marked likely to be rmeoved.